app/src/main/java/ch/epfl/sdp/blindly/settings/SettingsLocation.kt

```diff
@@ -20,7 +20,7 @@ class SettingsLocation : AppCompatActivity(), OnMapReadyCallback {
     private lateinit var mapView: MapView
     private var location: Location? = null
     private lateinit var locSer: LocationService
-    private val LAUSANNE = LatLng(46.5, 6.6)
+    private val LAUSANNE_LATLNG = LatLng(46.5, 6.6)

```

Comment:
I guess we are using Lausanne and EPFL as default values? Maybe we should merge them somehow, seems a little bit odd to me to have to different defaults.